src/kOS/Suffixed/VectorRenderer.cs

```diff
@@ -444,14 +444,16 @@ public void RenderPointCoords()
                 useWidth = (float)(Width * Scale * mapWidthMult);
 
                 // Position the arrow line:
-                line.SetVertexCount(2);
-                line.SetWidth(useWidth, useWidth);
+                line.positionCount = 2;
+                line.startWidth = useWidth;
+                line.endWidth = useWidth;
```

Comment:
Fixes Warnings: obsoleted methods - SetVertex and SetWidth.

Unity changed the API a bit here.